SPECS/eventlog/eventlog.spec

```diff
@@ -0,0 +1,60 @@
+Summary:	Syslog event logger library
+Name:		eventlog
+Version:	0.2.12
+Release:	1%{?dist}
+License:	GPL
+URL:		https://www.balabit.com
+Group:		System Environment/Daemons
+Vendor:		VMware, Inc.
+Distribution: 	Photon
+Source0:	https://www.balabit.com/downloads/files/eventlog/0.2/eventlog_0.2.12.tar.gz
```

Comment:
Can you replace eventlog_0.2.12.tar.gz with name and version tags as in other spec files.